internal/daemon/worker/controller_connection.go

```diff
@@ -35,6 +35,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const hcpb_url_suffix = ".proxy.boundary.hashicorp.cloud"
```

Comment:
I realize this is a common const style but I think our normal const style is also camelCase